src/model/SubSurface.hpp

```diff
@@ -176,6 +176,7 @@ namespace model {
     void resetAdjacentSubSurface();
 
     /** Returns the SurfacePropertyConvectionCoefficients, if it exists. */
+    // cppcheck-suppress [duplInheritedMember] because PlanarSurface is dumb and returns a vector
     boost::optional<SurfacePropertyConvectionCoefficients> surfacePropertyConvectionCoefficients() const;
```

Comment:
I didn't want to break API, but this is clearly wrong.

This is wrong: https://github.com/NREL/OpenStudio/blob/62f977e804d9b6f6580dbedf760de621e2abf2ec/src/model/PlanarSurface.hpp#L199-L200